api/core/tools/provider/builtin/spider/tools/scraper_crawler.yaml

```diff
@@ -74,7 +74,6 @@ parameters:
     human_description:
       en_US: blacklist a set of paths that you do not want to crawl. you can use regex patterns to help with the list.
       zh_Hans: 指定一组不想爬取的路径。您可以使用正则表达式模式来帮助定义列表。
-    placeholder: /blog/*, /about
```

Comment:
You can leave this placeholder, but please use the same i18n format as the label.